contracts/src/challenge/SymChallenge.sol

```diff
@@ -55,26 +56,25 @@ contract SymChallenge is ChallengeBase, ISymChallenge {
     }
 
     /**
-     * @notice Initializes contract.
+     * @notice Initializes contract. Called by `Rollup.sol` when a challenge is initiated. Challenge requires additional inputs from defender and challenger to be executed.
      * @param _defender Defending party.
      * @param _challenger Challenging party. Challenger starts.
      * @param _verifier Address of the verifier contract.
      * @param _daProvider DA provider.
      * @param _resultReceiver Address of contract that will receive the outcome (via callback `completeChallenge`).
      * @param _startStateHash Commitment to agreed-upon start state.
-     * @param _endStateHash Commitment to disagreed-upon end state being challenged.
      */
+
     function initialize(
         address _defender,
         address _challenger,
         IVerifier _verifier,
         IDAProvider _daProvider,
         IChallengeResultReceiver _resultReceiver,
         bytes32 _startStateHash,
-        bytes32 _endStateHash,
```

Comment:
one issue here is a malicious defender can propose the correct hash—which isn't consistent with what the assertion committed to, but they nevertheless are able to win the challenge